packages/engine/lib/stateful/src/message/batch.rs

```diff
@@ -36,11 +36,14 @@ const UPPER_MULTIPLIER: usize = 1000;
 /// Size of shared memory above which the soft upper bound is checked
 const LOWER_BOUND: usize = 10000;
 
-/// A group of messages.
+/// A list of [`Message`].
```

Comment:
```suggestion
/// A collection of [`Message`]s.
```
Or would you prefer we don't use the term collection?

Reply:
I also thought about "collection" but thought that `list` would be more specific. I tried to avoid writing "A batch of `Message`s" to describe `MessageBatch`. 😄